kano_updater/utils.py

```diff
@@ -134,23 +127,19 @@ def run_for_every_user(cmd):
     for user in get_users():
         run_cmd_log("sudo su -c '{cmd}' - {user}".format(cmd=cmd, user=user))
 
-
 def is_server_available():
-    install_ping()
-    import ping
-
-    lost_packet_percent = ping.quiet_ping(REPO_SERVER, timeout=3, count=5)[0]
-
-    return lost_packet_percent < 50
+    """ Pings REPO_SERVER to diagnose packet loss
+    and server availability.
 
+    Args:
+        REPO_SERVER: is the pre-set address
+        count: Number of times to ping the server
 
-def install_ping():
-    try:
-        import ping
```

Comment:
Unless I'm missing something, ping doesn't seem to be imported now?

Reply:
Thanks, chopped off too much.